common-arcana.lic

```diff
@@ -456,7 +458,10 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'], command)
+    tm_tattoo = false
+    tm_tattoo = data['tattoo_tm'] if data['tattoo_tm']
```

Comment:
Ruby treats nil as falsey. Can you simplify lines 461 and 462 to `tm_tattoo = data['tattoo_tm']` (or even in-line it on line 464)?

Reply:
I did it the way above to avoid nils. So, it will set false, if the data has it's own setting it will then set it to whatever it was, Then it will go through. Where is the nil? I am just missing something, I did originally have it 
```prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, data['tattoo_tm'])```
but figured if that line was nill, it's a shitshow. I am just missing something.

Reply:
If `data['tattoo_tm']` is nil, the `if` inside of `prepare?` will evaluate to false

Reply:
NM, I realized my issue when I finally woke.